assets/aml-benchmark/components/inference-postprocessor/spec.yaml

```diff
@@ -4,7 +4,7 @@ type: command
 name: inference_postprocessor
 display_name: Inference Postprocessor
 description: Inference Postprocessor
-version: 0.0.1
+version: 0.0.1.1
```

Comment:
Let's remove the component version update. When we release a component, we will update it then.

Reply:
okay. Done.